bootstrap-page.html

```diff
@@ -0,0 +1,173 @@
+                <button type="button">Explore the docs</button>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <!-- grey border around the first two <p> and another border aound the last two <p>-->
+        <div class="row">
+            <div class="column">
+                <!-- both <p> have a button to the right -->
+                <!-- add a class so links do not open but can have starting and ending tags as normal text -->
+               <p>CSS only</p>
+               <p>link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous"</p>
+               <p>JavaScript Bundle with Popper</p>
+               <p>script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW" crossorigin="anonymous">script</p>
+            </div>
+        </div>
+    </div>
+    <div class="container-3">
+        <div class="row">
+            <div class="column">
+                <img src="url" alt="purple box with white circle and a square that is missing one side">
+                <h2>Bootstrap Icons</h2>
+                <h4>For the first time ever, Bootstrap has its own open source SVG icon library, designed to work best with our components and documentation.</h4>
+                <p>Bootstrap Icons are designed to work best with Bootstrap components, but they’ll work in any project. They’re SVGs, so they scale quickly and easily, can be implemented in several ways, and can be styled with CSS.</p>
+                <button type="button">Get Bootstrap Icons</button>
+            </div>
+        </div>
+    </div>
+    <div class="container-2">
+        <div class="row">
+            <div class="column">
+                <img src="#" alt="picture of different bootstrap icons">
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <img src="#" alt="red box with white tear drop">
+                <h1>Official Themes</h1>                
+                <h4>Take Bootstrap to the next level with premium themes from the<a href="https://themes.getbootstrap.com/">official Bootstrap Themes marketplace</a></h4>
+                <p>Themes are built on Bootstrap as their own extended frameworks, rich with new components and plugins, documentation, and powerful build tools.</p>
+                <button type="button">Browse themes</button>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <img src="#" alt="picture of different bootstrap themes">
+            </div>
+        </div>
+    </div>
+    <footer>
+        <div class="footer">
+            <div class="row">
```

Comment:
You'd want a top level `div` with class `container` here as well...
rows, and column's need to be inside of a container `div` usually

Reply:
The footer should be in a div?

Reply:
You already have a div on line 157.
This is where container class would go.
You cannot have rows and columns without a top level container, usually with a class of container.

Reply:
I have a class “footer”

Reply:
This is almost correct.
In nearly all cases, you have used a container class, which houses a row class, which houses various column classes.  In this case, you also need that structure.  So on line 172, add the class of `container` to be consistent.  You can also leave the class of `footer` in there as well, as any element can and often has multiple classes.

Reply:
@blam-codes - adding another comment to check for spam

Reply:
@blam-codes